browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2520,6 +2541,39 @@ L.CanvasTileLayer = L.Layer.extend({
 		target.onclick = target.ontouchend = func;
 	},
 
+	_getFunctionList: function(textMsg) {
+		var maxSuggestion = 3;
+		var functionNameList = [];
+		var resultText = '';
+		var currentFuncDescription = '';
+
+		var suggestionArray = JSON.parse(textMsg);
+		if (suggestionArray.length < maxSuggestion) { maxSuggestion = suggestionArray.length; }
+
+		for (var i = 0; i < maxSuggestion; i++) {
+			if (i == 0)
+				currentFuncDescription = suggestionArray[i].description;
+
+			var signature = suggestionArray[i].signature;
+			functionNameList.push(signature.substring(0,signature.indexOf('(')));
+		}
+
+		for (var i = 0; i < maxSuggestion; i++) {
+			if (i == 0)
+				resultText = resultText + '[' + functionNameList[i] + ']';
+			else
+				resultText = resultText + ', ' + functionNameList[i];
+		}
+
+		var remainingFuncCount = suggestionArray.length - maxSuggestion;
+		if (remainingFuncCount > 0)
+			resultText = resultText + ' and ' + remainingFuncCount + ' more';
```

Comment:
now I noticed not-translated strings here...
It should be:
`resultText + ' ' + _('and %COUNT more').replace('%COUNT, remainingFuncCount)`

(no spaces at the ends of translated string)

Reply:
good catch!

Reply:
Done thanks